orc8r/tools/ansible/roles/docker/tasks/install_debian.yml

```diff
@@ -29,9 +29,14 @@
       - software-properties-common
       - python3-pip
 
+- name: Copy docker gpg key from codebase
+  copy:
+    src: docker.key
+    dest: /tmp/docker.key
+
 - name: Add Docker’s official GPG key
   apt_key:
-    url: https://download.docker.com/linux/ubuntu/gpg
+    file: /tmp/docker.key
     state: present
 
 - name: Verify that you now have the key with the fingerprint
```

Comment:
NIT: do we still need to verify vs a fingerprint if the key is checked in?

Reply:
done